05/app.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { createRoot } from "react-dom/client";
+
+const root = createRoot(document.querySelector("#root"));
+
+class WeatherCheck extends React.Component {
+  constructor(props) {
+    super(props);
+    const {lat} = this.props;
```

Comment:
Potrzebujemy pobrać obie współrzędne przez props wiec powinno być: `const {lat, lot} = this.props;`